selfapi.js

```diff
@@ -6,6 +6,11 @@ var https = require('https');
 var nodepath = require('path');
 var url = require('url');
 
+var options = {
```

Comment:
Or you could also keep your `var options = {` unchanged, and add `exports.options = options;` somewhere, e.g. at the bottom of the file.